pkg/actions/addon/addon.go

```diff
@@ -118,19 +112,6 @@ func (a *Manager) getLatestMatchingVersion(addon *api.Addon) (string, error) {
 	return versions[0].Original(), nil
 }
 
-func supportedVersion(version string) error {
-	supported, err := utils.IsMinVersion(api.Version1_18, version)
-	if err != nil {
-		return err
-	}
-	switch supported {
-	case true:
-		return nil
-	default:
-		return fmt.Errorf("addons not supported on %s. Must be using %s or newer", version, api.Version1_18)
```

Comment:
why are we removing this check? shouldn't this be 
`return fmt.Errorf("addons not supported on %s. Must be using %s or newer", version, api.Version1_19)`

Reply:
same answer as below